lib/facter/db_is_master.rb

```diff
@@ -0,0 +1,5 @@
+Facter.add('db_is_master') do
+  setcode do
+    Facter::Core::Execution.exec('sudo -u postgres psql -t aristamd -c "select pg_is_in_recovery();" | /bin/grep -qi "f" && echo "true" || echo "false"')
```

Comment:
What is `aristamd`?

Reply:
In this case?  An artifact I forgot to take out... sorry about that!